packages/wallet/tsconfig.json

```diff
@@ -19,17 +21,15 @@
     "resolveJsonModule": true,
     "esModuleInterop": true
   },
+  "include": [
+    "src/**/*",
+    "types/globals.d.ts",
+    "contracts/**/*.json"
+  ],
+  "references": [
+    { "path": "../nitro-protocol"}
+  ],
   "exclude": [
-    "node_modules",
```

Comment:
All of these should still be excluded, no?

Reply:
They are excluded because they are not included except for `src/**/*`